google-style.xml

```diff
@@ -46,19 +46,16 @@
             <property name="ignorePattern" value="^package.*|^import.*|a href|href|http://|https://|ftp://"/>
         </module>
         <module name="AvoidStarImport">
-			<property name="severity" value="warning"/>
-		</module>
+            <property name="severity" value="warning"/>
+        </module>
         <module name="OneTopLevelClass"/>
         <module name="NoLineWrap"/>
         <module name="EmptyBlock">
             <property name="option" value="TEXT"/>
             <property name="tokens" value="LITERAL_TRY, LITERAL_FINALLY, LITERAL_IF, LITERAL_ELSE, LITERAL_SWITCH"/>
         </module>
         <module name="NeedBraces"/>
-        <module name="LeftCurly">
-            <property name="maxLineLength" value="100"/>
```

Comment:
Was it intentionally removed?

Reply:
yes, it was deprecated for a long time and finally it was removed in [Checkstyle 8.2](http://checkstyle.sourceforge.net/releasenotes.html#Release_8.2): https://github.com/checkstyle/checkstyle/issues/3671